velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -146,6 +146,43 @@ class ArithmeticTest : public SparkFunctionBaseTest {
     return evaluateOnce<double>("divide(c0, c1)", numerator, denominator);
   }
 
+  template <typename T>
+  std::optional<T> checkAdd(std::optional<T> a, std::optional<T> b) {
+    return evaluateOnce<T>("check_add(c0, c1)", a, b);
+  }
+
+  template <typename T>
+  std::optional<T> checkSubstract(std::optional<T> a, std::optional<T> b) {
+    return evaluateOnce<T>("check_subtract(c0, c1)", a, b);
+  }
+
+  template <typename T>
+  std::optional<T> checkMultiply(std::optional<T> a, std::optional<T> b) {
+    return evaluateOnce<T>("check_multiply(c0, c1)", a, b);
+  }
+
+  template <typename T>
+  std::optional<T> checkDivide(std::optional<T> a, std::optional<T> b) {
+    return evaluateOnce<T>("check_divide(c0, c1)", a, b);
+  }
```

Comment:
I wonder if we can combine the four functions by using functionName as a parameter.

Reply:
Updated, thanks.